pkg/cluster/spec/spec.go

```diff
@@ -515,6 +516,17 @@ func (s *Specification) validateTiSparkSpec() error {
 		return ErrMultipleTiSparkMaster
 	}
 
+	// Multiple workers on the same host is not supported by Spark
+	if len(s.TiSparkWorkers) > 1 {
+		cnt := make(map[string]int)
+		for _, w := range s.TiSparkWorkers {
+			if cnt[w.Host] > 0 {
+				return ErrMultipleTisparkWorker
```

Comment:
It's better to attach the `host` to the error message to help users figure out which host is duplicated.